Sources/Prometheus/MetricTypes/Histogram.swift

```diff
@@ -1,8 +1,11 @@
+import NIOConcurrencyHelpers
+
 /// Default buckets used by Histograms
 public var defaultBuckets = [0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1.0, 2.5, 5.0, 7.5, 10.0, Double.greatestFiniteMagnitude]
 
 /// Label type Histograms can use
 public protocol HistogramLabels: MetricLabels {
+    /// Bucket
     var le: String { get set }
```

Comment:
is this `le` on purpose?

Reply:
Yup, requirement set forth by prometheus. The value in the label HAS to be called `le` in Histograms